mne/epochs.py

```diff
@@ -1350,7 +1350,7 @@ def copy(self):
         return new
 
     @verbose
-    def save(self, fname, split_size='2GB', fmt='single', verbose=True):
+    def save(self, fname, split_size='2GB', fmt='single', overwrite=False, verbose=True):
```

Comment:
This would be a change of behavior, no? Previously, people who used `epochs.save` consecutively on their scripts would not have their scripts running anymore

Reply:
Yes true I guess we need to properly deprecate the old behavior with a warning for one release cycle.

Reply:
yeah, although you'd have to catch all the deprecation warnings in the tests then.

Reply:
Yes this is the pain we have to deal with anytime we deprecate. Though usually we work around it by passing explicitly the parameter (here it would be `overwrite=True`) to get the old behavior and have no deprecation warning.

Reply:
I did this in all the relevant unit tests that I could find, per my description above:

"I set the default to be overwrite=False, for consistency with raw.save. This changes the behavior of epochs.save. Is this correct or should I add a deprecation warning?

I had to add overwrite=True as an argument to epochs.save in a number of unit tests, so that the behavior would be consistent with previous versions of epochs.save. Please also let me know if this is appropriate."

Not sure if this is sufficient?

Reply:
That sounds like the correct approach

Reply:
... the `overwrite=True` added to tests is correct, but the deprecation itself currently is not. You should do:
```
... overwrite=None):

    if overwrite is None:
        overwrite = True
        warn('overwrite defaults to True in 0.18 but will change to False in 0.19, '
              'set it explicitly to avoid this warning', DeprecationWarning)
```

Reply:
Thanks!